### ISSUE TITLE
Spot instances support

### ISSUE THREAD

**Issue by lielran** [machulav/ec2-github-runner#5]
really like the idea of on-demand runners. 
spot support will make it more appealing :)

**Comment by machulav** [comment]
Thank you for the feedback and the idea contribution - very appreciate it!

The idea of using spot instances is interesting.
I will definitely take a look into it.

**Comment by lielran** [comment]
also, be aware that in order to communicate back to github the ec2 needs internet access via public IP or internet gateway

**Comment by machulav** [comment]
@lielran, you're right! Thank you for the note. I will add this to the documentation.

**Comment by ravishankar-nmlz** [comment]
Any update on this, this will be a very helpful cost reducing workflow

**Comment by nocodehoarder** [comment]
Any update on this? This is one awesome cost saving enhancement.

**Comment by AbdealiJK** [comment]
I'm looking for spot instances too - any plans on implementing this (and timelines?)

I see that EC2 launch templates has a open PR too

**Comment by sujithpaily** [comment]
+1
Looking for same

**Comment by Preen** [comment]
No updates on this?

**Comment by Preen** [comment]
Closing this as we've released this feature.
